src/main/java/com/cbfacademy/App.java

```diff
@@ -10,33 +10,55 @@
 import java.util.List;
 
 import com.cbfacademy.Employee.Employee;
+import com.cbfacademy.FileHandler.JSONFileHandler;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
 
 public class App {
     public static void main(String[] args) {
-        
-        public static List<Employee> readFile(String filePath) {
-           
-        Path filePath = Paths.get("C:\\Users\\Admin\\cbfacademy\\java-exercises-YOkolo24\\java-json-exercise-YOkolo24\\src\\main\\java\\com\\cbfacademy\\Employee\\Employee.java");
-        
-        try {
-            BufferedReader input = new BufferedReader(new FileReader(filePath));
-                List<String> lines = Files.readAllLines(filePath2);
 
-                // File sourceFile = Paths.get(filePath).toFile();
-                // ObjectMapper objectMapper = new ObjectMapper();
-                // List<Employee> employees = Arrays.asList(objectMapper.readValue(sourceFile, Employee[].class));
+        JSONFileHandler newJsonFileHandler = new JSONFileHandler();
 
-                // employees.forEach(System.out::println);
-
-            } catch (IOException e) {
-                System.err.println(e.getMessage());
-            }
+        Path filePath = Paths.get("C:\\Users\\Admin\\cbfacademy\\java-exercises-YOkolo24\\java-json-exercise-YOkolo24\\src\\main\\resources");
+                
+        try {
+            List<Employee> employees = readFile(filePath);
+            Gson gson = new Gson();
+            List<List<Employee>> employeesList = Arrays.asList(((Object) employees).readValue(Employee[].class));
+            String json = gson.toJson(employeesList);
```

Comment:
Once you're calling the method which does the reading from a JSON file and returns a list, you shouldn't need this here. You also shouldn't need a nested list.